### ISSUE TITLE
Regression: endless loop in helix up

### ISSUE THREAD

**Issue by tripodsan** [adobe/helix-cli#958]
**Description**
page rendering does into endless loop with the following code:

```
<script src="<esi:include src="sdfsdf">"></script>
```

**Version:**
4.2.0

**Comment by stefan-guggisberg** [comment]
The following HTML snippet with a malformed (non-self closing) `esi:include` tag leads to an endless loop:

```html
<head>
  <script src="<esi:include src='foo'>"></script>
</head>
```

It's a bug in `nodesi`: https://github.com/Schibsted-Tech-Polska/nodesi/issues/20

**Comment by tripodsan** [comment]
@stefan-guggisberg do you work on a PR to fix the issue in nodesi? if so, please mention it in their issue...

**Comment by stefan-guggisberg** [comment]
fixed in `nodesi` v1.11.0 -> `helix-simulator` v2.12.20 -> `helix-cli` v4.9.12
